src/outputs/MFEMConduitDataCollection.C

```diff
@@ -0,0 +1,25 @@
+#include "MFEMConduitDataCollection.h"
+
+registerMooseObject("PlatypusApp", MFEMConduitDataCollection);
+
+InputParameters
+MFEMConduitDataCollection::validParams()
+{
+  InputParameters params = MFEMDataCollection::validParams();
+  params.addClassDescription("Output for controlling MFEMConduitDataCollection inherited data.");
+  params.addParam<std::string>("protocol",
+                               "hdf5",
+                               "Conduit relay I/O protocol to use. Options: hdf5 (default), json, "
+                               "conduit_json, conduit_bin");
```

Comment:
Given there's a finite set of allowable options here, it's probably worth using an enum for checking that the user input falls inside this set, rather than just allowing any string

Reply:
I was thinking about whether to make it an enum. In the end, because the underlying object that gets passed to the MFEM Conduit data class is a string and not an enum, I opted to align the data types, like we did with the device choice. Still, we can make it an enum, no problemm